src/schematools/importer/base.py

```diff
@@ -498,6 +525,7 @@ def index_factory(
     db_table_name: Optional[str] = None,
     db_schema_name: Optional[str] = None,
     logger: Optional[LogfileLogger] = None,
+    db_table_temp_name: Optional[Dict] = {},
```

Comment:
Tip voor annotations: `Optional[Dict[str, str]]` of `Optional[dict]`.

Ter info, die `Dict` het hoofdletters is een speciale uitvinding omdat `dict[str, str]` niet mogelijk was in Python. In Python 3.9 hebben ze een oplossing gevonden en kunnen we zelfs `dict[str, str] | None` schrijven.